views/pulsar/v2/helpers/form.html.twig

```diff
@@ -901,33 +902,67 @@ show-label  | bool   | Control visibility of the `<label>` element without affec
 
 ## Options applied to input
 
-Option      | Type   | Description
------------ | ------ | ---------------------------------------------------------
-autofocus   | bool   | Whether the field should have input focus on page load
-disabled    | bool   | Stops the element from being interactive if true
-form        | string | Specific one or more forms this label belongs to
-format      | string | Options for this can be `default`, `US` or `reverse`
-id          | string | A unique identifier, will also be used as the label's `for` attribute
-name        | string | The name of this control
-placeholder | string | A short hint that describes the expected value
-required    | bool   | Adds `required` attribute
-value       | string | Specifies the value of the input
-data-*      | string | Data attributes, eg: `'data-foo': 'bar'`
+Option                | Type   | Description
+--------------------- | ------ | ---------------------------------------------------------
+autofocus             | bool   | Whether the field should have input focus on page load
+disabled              | bool   | Stops the element from being interactive if true
+form                  | string | Specific one or more forms this label belongs to
+format                | string | Options for this can be `default`, `US` or `reverse`
+id                    | string | A unique identifier, will also be used as the label's `for` attribute
+name                  | string | The name of this control
+placeholder           | string | A short hint that describes the expected value
+required              | bool   | Adds `required` attribute
+value                 | string | Specifies the value of the input
+data-*                | string | Data attributes, eg: `'data-foo': 'bar'`
+data-pulsar-datepicker | bool   | If `'true'`, the more accessible pulsar-date-picker will be used instead of pikaday
 
 *Any other options not listed here will be applied to the input.
 
 #}
 {% macro date(options) %}
 {% spaceless %}
     {% import _self as form %}
+    {% import '@pulsar/pulsar/v2/helpers/html.html.twig' as html %}
+
+    {# check for use of pulsar-date-picker #}
+    {% if options['data-pulsar-datepicker'] is defined and options['data-pulsar-datepicker'] is not empty and options['data-pulsar-datepicker'] == true %}
+
+        {# Generate guid to link input to trigger button, required for pulsar-date-picker #}
+        {% set guidDatePickerTriggerGuid = 'guid-' ~ random() %}
```

Comment:
Minor, but the duplication of guid in the var could be simplified? `datePickerTriggerGuid` ?

Reply:
Yep typo, good spot, fixed